lib/rubocop/cop/style/block_delimiters.rb

```diff
@@ -248,6 +286,25 @@ def proper_block_style?(node)
           end
         end
 
+        def special_method?(node)
+          method_name = node.method_name
+          ignored_method?(method_name) || braces_required_method?(method_name)
+        end
+
+        def special_method_proper_block_style?(node)
+          method_name = node.method_name
+          return true if ignored_method?(method_name)
+          return node.braces? if braces_required_method?(method_name)
+        end
```

Comment:
Seems like we pass around `node` quite a bit to methods that only need the `method_name`.

Reply:
We need to do `node.braces?` in `special_method_proper_block_style?`, but changed the param to `special_method?` to just `method_name`.

https://github.com/rubocop-hq/rubocop/commit/44c37b6fcf232cd7c401c80dbdc0bf41f36d1ad7